go/database/mpt/io/archive.go

```diff
@@ -67,7 +72,7 @@ func ExportArchive(ctx context.Context, logger *Log, directory string, out io.Wr
 	}
 
 	logger.Printf("opening archive: %s", directory)
-	archive, err := mpt.OpenArchiveTrie(directory, info.Config, mpt.NodeCacheConfig{}, mpt.ArchiveConfig{})
+	archive, err := mpt.OpenArchiveTrie(directory, info.Config, nodeConfig, mpt.ArchiveConfig{})
```

Comment:
Let's add the `ArchiveConfig` as well to the parameters, sooner or later somebody will want to configure this as well.